docs/jvs-ui.md

```diff
@@ -2,7 +2,7 @@
 
 **JVS is not an official Google product.**
 
-[JVS UI](../cmd/ui) facilitates the justification verification flow using a UI
+[JVS UI](../cmd/ui) facilitates the justification verification flow using a UI. It is meant to be an alternative to using `jvsctl` which requires integrating a calling application that is non-customer facing.
```

Comment:
It's probably better to have this in a separate PR

Reply:
yeah didnt mean to bring it in yet, redacting until we get those screenshots.